whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/RoleSpec.groovy

```diff
@@ -671,6 +671,43 @@ class RoleSpec extends BaseQueryUpdateSpec {
         query_object_not_matches("-rBT role FR1-TEST", "role", "First Role", "tech-c:")
     }
 
+    def "punycode abuse-mailbox with IDN"() {
```

Comment:
Need an additional test to confirm local-part with a non-ASCII value is *not* converted to Punycode but is rejected with a syntax error.

Reply:
Confirmed that IDNEmailAddressConverter only converts the domain part, not the local part 👍